statik/utils.py

```diff
@@ -10,6 +10,7 @@
 from copy import deepcopy, copy
 import shutil
 
+import yaml
```

Comment:
Why is this being imported? I don't see it being used anywhere in this file.

Reply:
Hehe, thanks. Earlier, the autogen function was in utils.py and when I moved it to autogen.py I forgot to remove this line.